Dockerfile

```
@@ -85,7 +95,7 @@ RUN echo "source $CCTL/activate" >> .bashrc
 COPY --chown=cctl:cctl ./docker/start.sh .
 RUN chmod +x start.sh
 
-EXPOSE 11101-11105 14101-14105 18101-18105
+EXPOSE 11101-11105 14101-14105 18101-18105 25101-25101
 
 HEALTHCHECK --interval=10s --timeout=5s --retries=4 --start-period=20s \
     CMD curl --silent --location 'http://127.0.0.1:11101/rpc' --header 'Content-Type: application/json' --data '{"id": "1", "jsonrpc": "2.0", "method": "info_get_status", "params": []}' | jq -e -n 'input.result.reactor_state' | grep "Validate"
```

RPC is now available at port `21101`, so healthcheck must be updated:

```suggestion
    CMD curl --silent --location 'http://127.0.0.1:21101/rpc' --header 'Content-Type: application/json' --data '{"id": "1", "jsonrpc": "2.0", "method": "info_get_status", "params": []}' | jq -e -n 'input.result.reactor_state' | grep "Validate"
```

---

Fixed in 686f53fce12a6fd2e7b7590430eaf8902c22f01e.